pkgs/os-specific/linux/fw-ectool/default.nix

```diff
@@ -1,38 +1,43 @@
 { stdenv
 , lib
-, fetchFromGitHub
+, fetchFromGitLab
+, cmake
 , pkg-config
-, hostname
+, libusb1
+, libftdi1
 }:
 
 stdenv.mkDerivation {
   pname = "fw-ectool";
-  version = "unstable-2022-12-03";
+  version = "0-unstable-2023-12-15";
```

Comment:
why do you prefix this with `0-`? it seems to work for me without it too.

Reply:
Due to the new versioning scheme documented in https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming